feat: Add email sending feature (#4)
algolia-samples/email-recommendations: README.md

ALGOLIA_API_KEY=<replace-with-your-algolia-api-key>
```

### 4. (Optional) Configure an email sending provider

> The sample application allow you to send the preview to your email address using an email sending provider.

Reviewer: ```suggestion
With this sample app, you can send the preview to your email address.
```

> We do provide a minimal working exemple with [SendGrid](https://sendgrid.com/).

Reviewer: ```suggestion
We provide a minimal working example with [SendGrid](https://sendgrid.com/) as email provider.
```
